lib/parsers/index.ts

```diff
@@ -55,7 +55,42 @@ function buildSubTree(dep): PkgTree {
   return depSubTree;
 }
 
-export async  function parseManifestFile(manifestFileContents: string) {
+export async function getDependencyTreeFromPackageReference(manifestFile) {
+  const packageList = _.get(manifestFile, 'Project.ItemGroup', [])
+    .find((itemGroup) => _.has(itemGroup, 'PackageReference'));
+
+  const depTree: PkgTree = {
+    dependencies: {},
+    hasDevDependencies: false,
+    name: '',
+    version: '',
+  };
+
+  if (!packageList) {
+    return depTree;
+  }
+
+  for (const dep of packageList.PackageReference) {
+    const depName = dep.$.Include;
+    depTree.dependencies[depName] = buildSubTreeFromPackageReference(dep);
+  }
+
+  return depTree;
+}
+
+function buildSubTreeFromPackageReference(dep): PkgTree {
+
+  const depSubTree: PkgTree = {
+    depType: DepType.prod,
```

Comment:
I believe we can use `DevelopmentDependencies` https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference